code/src/index.css

```diff
@@ -1,13 +1,23 @@
+@import url('https://fonts.googleapis.com/css2?family=Quicksand:wght@300;400;500;600;700&display=swap');
+
+:root {
+  --lax: #e59089;
+  --white: #fff4f3;
+  --brown: #9b0c33;
+}
```

Comment:
I have never used that approach, what does it do? Setting root colors to the page?

Reply:
I am defining there some colors that I am using more often on my page. I can give for example a button, a div and paragraph the color: var(--lax). When I then later on decide, that I want to change the color to another value, I do this just on time in the :root and all the elements get the new coloring immediately